style.css

```diff
@@ -0,0 +1,50 @@
+body{
```

Comment:
Before the opening brace put one space.
See more in this guide: http://codeguide.academy/html-css.html#html-language